scripts/github-releases.js

```diff
@@ -0,0 +1,112 @@
+const assert = require("assert");
```

Comment:
It seems like these scripts are generic and generally usable and should go into the CI repo?

Reply:
@kraenhansen @emanuelez @alebsack @nirinchev ?

Reply:
I agree 👍 - but I wanted to implement them here and ensure they work before moving them.
BTW - this is the PR that merges these changes into master: https://github.com/realm/realm-studio/pull/1069